pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotSchemaRestletResource.java

```diff
@@ -251,14 +249,27 @@ public String validateSchema(FormDataMultiPart multiPart) {
       @ApiResponse(code = 500, message = "Internal error")
   })
   public String validateSchema(Schema schema) {
+    validateSchemaInternal(schema);
+    return schema.toPrettyJsonString();
+  }
+
+  private void validateSchemaName(Schema schema) {
+    if (StringUtils.isBlank(schema.getSchemaName())) {
+      throw new ControllerApplicationException(LOGGER,
+          "Invalid schema: " + schema.getSchemaName() + ". Reason: 'schemaName' should not be null",
```

Comment:
This exception message is not very readable as schema name is always blank. Consider changing it to `'schemaName' must be provided`